must-gather/collection-scripts/gather_nodes

```diff
@@ -62,6 +62,7 @@ do
     oc exec $pod -n perf-node-gather -- lspci -nvv > $NODE_PATH/lspci
     oc exec $pod -n perf-node-gather -- lscpu -e > $NODE_PATH/lscpu
     oc exec $pod -n perf-node-gather -- cat /proc/cmdline > $NODE_PATH/proc_cmdline
+    oc exec $pod -n perf-node-gather -- dmesg $NODE_PATH/dmesg.json
```

Comment:
missing shell redirect. Also, the output is not JSON

Reply:
fast push, fixed